### PR TITLE
[tests-only][full-ci]changing given from ocs to graph in feature file

### PR DESCRIPTION
**Pull request by S-Panta**
## Description
updating given statement of test suite to use graph API
Suites covered:
- coreApiShareManagementBasicToShares/createShareToSharesFolder.feature
- coreApiShareManagementBasicToShares/deleteShareFromShares.feature
Reva Edge bump : https://github.com/cs3org/reva/pull/4682
## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8717

## How Has This Been Tested?

- test environment:
- locally
- ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature [hunk 1]
```diff
@@ -377,14 +402,18 @@ Feature: sharing
     And group "grp1" has been created
     And user "Brian" has been added to group "grp1"
     And user "Alice" has uploaded file with content "Random data" to "/randomfile.txt"
-    And user "Alice" has shared file "randomfile.txt" with group "grp1"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | randomfile.txt |
+      | space           | Personal       |
+      | sharee          | grp1           |
+      | shareType       | group          |
+      | permissionsRole | Viewer         |
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature:410]: the scenario title and the steps do not match:
1. group name must have a capital letter
2. share should be an action (When step)

File: tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature [hunk 2]
```diff
@@ -394,19 +423,19 @@ Feature: sharing
...
     And user "Alice" has created folder "/PARENT"
     And user "Alice" has uploaded file with content "file in parent folder" to "/PARENT/parent.txt"
-    When user "Alice" shares folder "/PARENT" with group "😀 😁" using the sharing API
-    Then the OCS status code should be "<ocs-status-code>"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | PARENT   |
+      | space           | Personal |
+      | sharee          | 😀 😁    |
+      | shareType       | group    |
+      | permissionsRole | Viewer   |
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature:431]: this step should not be changed

File: tests/acceptance/features/coreApiShareManagementBasicToShares/deleteShareFromShares.feature [hunk 3]
```diff
@@ -149,42 +196,59 @@ Feature: sharing
...
+      | /shared                 | Editor          | 1               | 200              | /Shares/shared          |
+      | /shared                 | Editor          | 2               | 404              | /Shares/shared          |
 
 
   Scenario Outline: individual share recipient tries to delete the share
     Given using OCS API version "<ocs-api-version>"
     And user "Alice" has created folder "/shared"
     And user "Alice" has moved file "/textfile0.txt" to "/shared/shared_file.txt"
     And user "Alice" has shared entry "<entry-to-share>" with user "Brian"
+    And user "Alice" has sent the following resource share invitation:
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiShareManagementBasicToShares/deleteShareFromShares.feature:224]:
```suggestion
    And user "Alice" has sent the following resource share invitation:
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature (hunk 1 above)
Comment (S-Panta) [reply tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature:410]: 1 should be invalid. it says username should be capital, not the group name.

**Review by saw-jan — APPROVED** [review]

**Pull request merged**